nextflow.config

```diff
@@ -61,7 +60,7 @@ params {
     // Schema validation default options
     validationFailUnrecognisedParams = false
     validationLenientMode            = false
-    validationSchemaIgnoreParams     = 'genomes,igenomes_base'
+    validationSchemaIgnoreParams     = 'fasta,fasta_fai,genomes,igenomes_base'
```

Comment:
there should be another way to deal with this, but ok for now

Reply:
Yeah, I'm not sure that actually has any effect since I still get:
```
WARN: Access to undefined parameter fasta -- Initialise it to a default value eg. params.fasta = some_value
```
Anyways, let's fix that in the template-update 2.14.1